### ISSUE TITLE
Not Responding after sleep on windows

### ISSUE THREAD

**Issue by Singleton06** [desktop/desktop#8004]
## Description
Github desktop can't be closed after windows has been sleeping.  This is the same issue as #4672 but it has been closed and the last person to make a comment on it was told to log another issue (and it appears that it didn't happen).
## Version
* GitHub Desktop: 2.1.0
* Operating system: Windows 10 Enterprise

## Steps to Reproduce
Just seems to be if I have had github up and my computer went to sleep.  There might be more to it, but I haven't noticed any trends.

### Expected Behavior
To either have it working, or at least allow me to terminate the task.

### Actual Behavior
Frozen, can't terminate the program.  Requires a restart every time.

## Additional Information
Additionally, when this happens the program cannot be terminated through task manager (or through commandline ran as an administrator).
![image](https://user-images.githubusercontent.com/2235110/61718790-4a1a1b00-ad29-11e9-9ee9-b9338c8df2c7.png)

Command Prompt ran as admin (When I ran it before it terminated 2 other instances, but these wouldn't terminate):
![image](https://user-images.githubusercontent.com/2235110/61718895-79c92300-ad29-11e9-801d-a1fb740fdd0a.png)

### Logs
No logs generated for today's date.  Last log was 3 days ago, but I don't want to attach that since it has internal URLs to my company, etc.

**Comment by tierninho** [comment]
@Singleton06 Thanks for filing. We have yet to pinpoint the exact cause of this issue as it happens sporadically and it is hard for the user to send over info at the time of the incident. 

We have speculated that it could be a general Electron issue with sleep/wake, but there is nothing concrete yet. 

When you get a chance, please send the Desktop version number and any (redacted) logs if possible. 

How often is this happening?

**Comment by tierninho** [comment]
Also, if the command prompts don't work, try:

`wmic process where "name='myprocessname.exe'" delete` or `taskkill /IM "process_name" /T /F`

A less than ideal workaround may be quitting Desktop before putting the computer to sleep. 🤕

**Comment by Rexogamer** [comment]
I tried to reproduce this by opening Desktop then putting my laptop to sleep and waking it back up but I found no issues.

**Comment by Singleton06** [comment]
I will try to keep an eye on when it happens.  I tend to close my laptop on Fridays and open it back up again on Mondays, so perhaps it's an issue with prolonged sleep?  I will keep an eye on it and let you know if that pattern develops.  

@tierninho - I tried taskkill, just not with the /T.  I'll have to give that a shot next time or the other command you gave.

I've also updated the version in the report.  I'll try to see if I can get you the logs but just cleaned up

**Comment by Singleton06** [comment]
Two logs.  The 23rd is when I had the issue, the 20th was the only log that was showing at the time of github desktop not being able to close.
[2019-07-23.desktop.production.log](https://github.com/desktop/desktop/files/3427281/2019-07-23.desktop.production.log)
[2019-07-20.desktop.production.log](https://github.com/desktop/desktop/files/3427307/2019-07-20.desktop.production.log)

**Comment by tierninho** [comment]
Thanks for logs, didn't see much to go on. Let us know if it happens again!

**Comment by Singleton06** [comment]
Happened again this week to me.  It was after the weekend where my computer had been put to sleep for a while.  I feel like that is likely what's going on.  When the computer is sleeping for a few days (perhaps disconnecting from different wifi networks, etc.) the issue arises.

**Comment by tierninho** [comment]
Yes, perhaps the wake up time of the computer and application is faster than the wake up of the wifi networks, etc. However, when I use my Windows machine after a long weekend, everything works fine. 🤔 

Does your Windows 10 Enterprise OS have any security protocols that might delay response time?

**Comment by tierninho** [comment]
I googled the overall problem and it seems to be a common Windows 10 thing. It seems Desktop (and you) are just a victim of the issue.

https://answers.microsoft.com/en-us/windows/forum/all/white-screen-after-sleep-windows-10/8615c1a3-fdbb-479d-a011-d56cc7f06633

**Comment by Singleton06** [comment]
Hm... I'm not sure if that's applicable.  It's not my entire computer.   All other applications (including spotify which I believe uses electron) work just fine.  It's just github desktop.

As for security protocols, I can't say for sure.  I'm not really that aware of the protocols and other security settings that are done.

**Comment by tierninho** [comment]
@Singleton06 Thanks for the additional color. Unfortunately, we haven't pinpointed the exact cause so no immediate fix right now. Thanks for your patience.

**Comment by Singleton06** [comment]
Not a problem, let me know if there is anything that you want me to try out.

**Comment by velocity23** [comment]
Having issues with this exact problem, started only after the most recent update. Reboot is the only thing that solves it. Windows 10 Pro build 19040.1.

**Comment by incyi** [comment]
Problem is still there, Windows 10 pro.
Microsoft Windows [Version 10.0.19044.1741]

**Comment by tidy-dev** [comment]
Closing this as we are unable to reproduce. Feel free to add to this issue if more details are uncovered.
